extmod/machine_mem.c

```diff
@@ -3,7 +3,7 @@
  *
  * The MIT License (MIT)
  *
- * Copyright (c) 2013, 2014 Damien P. George
+ * Copyright (c) 2013, 2014, 2015 Damien P. George
```

Comment:
Perhaps not. I did a little digging and came up with this:
http://www.contentious.com/2007/01/07/copyright-notice-is-the-year-really-necessary/

I think its really a call for @dpgeorge to make.